tests/devices/test_lightning_qubit.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for the accessibility of the Lightning-Qubit device
+"""
+import pennylane as qml
+import numpy as np
+
+
+def test_integration():
+    """Test that the execution of lightning.qubit is possible and agrees with default.qubit"""
```

Comment:
Curious if it is also possible to test the `default.qubit` fallback behaviour?

Reply:
Actually I guess this isn't very important, since it is being tested in the lightning tests.

Reply:
Yes I'm not sure how we'd do it on this side :thinking: It might be hard to find an instance that doesn't have a pre-compiled wheel of Lightning provided. So perhaps the approach could be to compile from source with a flag. But agreed, this should be fairly well tested already in Lightning.